.github/workflows/validate.yml

```diff
@@ -90,6 +90,8 @@ jobs:
            --check-links-ignore "https://jupytercon.com" \
            --check-links-ignore "https://www.netapp.com" \
            --check-links-ignore "https://github.com/[^/]+/?$" \
+           --check-links-ignore "https://sloan.org" \
+           --check-links-ignore "https://www.bloomberg.com" \
```

Comment:
This one actually does not work for me. It redirects to `https://bloomberg.com`

Reply:
for me, going to "bloomberg.com" redirects to "www.bloomberg.com" 🤷